app/ledger.js

```diff
@@ -695,6 +695,7 @@ var ledgerInfo = {
 
   // the desired contribution (the btc value approximates the amount/currency designation)
   btc: undefined,
+  btcPrice: undefined,
```

Comment:
Yes, @mrose17's updates in ledger-client v0.8.46 brought in by #3397 are really nice and obsolete the `btcPrice` field I had added :-)